client/package.json

```diff
@@ -39,5 +36,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "autoprefixer": "^9.8.6",
+    "postcss": "^7.0.36",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.4"
```

Comment:
We actually need these in the regular dependencies section, not just devDependencies. If we only have them here, our Heroku deployment will fail with errors like "can't find module tailwindcss"

Reply:
I only changed the version. If you noticed, lines 13-16 have been removed (version 8) and 41-43 replaced them (version 7). I followed this linke, https://tailwindcss.com/docs/installation#post-css-7-compatibility-build since there was some error. I can reverse if you want me to.

Reply:
That is great debugging! And the version is perfectly fine. The issue here is that these three dependencies need to be moved back up to lines 13 - 16 because they are `dependencies` the app needs to run, not just `devDependencies` needed to develop. I believe that guide on the Tailwind site is potentially mistaken. `npm install -D tailwind...` "-D" means dev dependency. What you'll want to run is `npm install --save tailwindcss@npm:@tailwindcss/postcss7-compat postcss@^7 autoprefixer@^9` Which will put those into the main dependencies section.